README.md

```diff
@@ -100,7 +108,39 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+#make Animal class
+class Animal
```

Comment:
👍 
One small issue: `simba` needed to be an instance of `Lion` instead of `Animal`
```ruby
simba = Lion.new("Simba")
```